app/models/concerns/scorecards/lockable.rb

```diff
@@ -7,17 +7,36 @@ module Scorecards::Lockable
     validate :locked_scorecard, on: :update
 
     def lock_access!
-      self.locked_at = Time.now.utc
+      self.completed_at = Time.now.utc
+      self.progress = "completed"
```

Comment:
@hengsokly please consider moving "completed" to scorecard status constant then use that constant instead of the raw value "completed" or "in_review" as sometimes we used more than one place so use a constant for a better modification.